sandbox/grist/useractions.py

```diff
@@ -691,13 +698,19 @@ def _updateColumnRecords(self, table_id, row_ids, col_values):
                if has_diff_value(values, 'colId', c.colId)}
 
     if renames:
+      # When a column rename has occurred, we need to update the corresponding references in
+      # formula, ACL rules and dropdown conditions.
+
       # Build up a dictionary mapping col_ref of each affected formula to the new formula text.
       formula_updates = self._prepare_formula_renames(renames)
 
       # For any affected columns, include the formula into the update.
       for col_rec, new_formula in sorted(six.iteritems(formula_updates)):
         col_updates.setdefault(col_rec, {}).setdefault('formula', new_formula)
 
+      acl.perform_acl_rule_renames(self, renames)
```

Comment:
@dsagal @SleepyLeslie Can I move those two methods down below (where it was before)? 
The new order of things generates acl rename actions before the column is actually renamed. Before the order was reversed, first `RenameAction` then `UpdateRecord` (for ACL)

Reply:
After discussing it offline, I will update it in another PR.